src/sdl/surface.cpp

```diff
@@ -17,6 +17,8 @@
 #include "color.hpp"
 #include "sdl/rect.hpp"
 
+const SDL_PixelFormat neutral_format = SDL_PIXELFORMAT_ARGB8888;
```

Comment:
Also unnecessary, can just use `SDL_PIXELFORMAT_ARGB8888` directly as before. If you're going to add a constant, it should be in the anon namespace.

Reply:
Updated.